src/index.js

```diff
@@ -0,0 +1,127 @@
+import './style.css';
+import setState from './modules/functionality.js';
+
+let tasks = [];
+const taskWrapper = document.querySelector('.to-dos');
+const newTask = document.querySelector('.new-task');
+const addNewTask = document.querySelector('.submit');
+const clearAll = document.querySelector('.clear-all');
+
+const addToLocalStorage = () => {
+  localStorage.setItem('myTasks', JSON.stringify(tasks));
+};
+
+const getFromLocalStorage = () => {
+  if (localStorage.getItem('myTasks')) {
+    tasks = JSON.parse(localStorage.getItem('myTasks'));
+  }
+  return tasks;
+};
+
+const resetIndex = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    let indx = i + 1;
+    indx = i;
```

Comment:
Hello @keza681 , everything looks perfect, just change the variable notation. Kindly make it descriptive